### ISSUE TITLE
Remove deprecated cycle models from Python?

### ISSUE THREAD

**Issue by jowr** [CoolProp/CoolProp#1081]
The cycle calculations in `CoolProp.Plots.SimpleCycles` have been deprecated for quite some time now. Since the whole plotting package is getting an overhaul, I would really like to remove `SimpleCycle`, `TwoStage` and `EconomizedCycle` from the code. This of course means that I will provide alternatives that use the new `StatePoint` objects. 

Any comments?

**Comment by ibell** [comment]
Perhaps email the mailing list to ascertain how many people are using them?

**Comment by davideziviani** [comment]
Hey guys! I think we could solve this cycle model issue and also part of  #832  by providing examples directly in TSTK for vapor compression and in ORCmKit for organic Rankine cycles and make use of CoolProp StatePoint

**Comment by davideziviani** [comment]
Hey guys!

I think we could solve this cycle models issue by providing examples directly in TSTK for vapor compression and in ORCmKit for Rankine cycles. This  would also partially solve #832

**Comment by jowr** [comment]
I am not sure that this would solve the issue. I already demonstrated the use of the cycle models and we have most of it running in CoolProp. See, http://www.coolprop.dreamhosters.com/binaries/sphinx/coolprop/python-cycles.html 

This issue is about *deleting* models, not about adding new ones or linking other packages.

**Comment by davideziviani** [comment]
@jowr  yes I understand but some people might still find those useful so you could remove them from CoolProp but place them somewhere else, for example in one of the projects I mentioned, where we actually use these models

**Comment by ibell** [comment]
How about just modifying the examples to use the new state class?  Given
the rather negative experience I have had with deprecating the Ts and Ph
diagrams without a stable replacement, I'm somewhat loathe to make more
non-backwards-compatible changes

**Comment by ibell** [comment]
Closing as part of an issue-tracker cleanup — this issue has had no activity in over two years.

*To be transparent: this comment and the close action are being performed by Claude (Anthropic's AI coding assistant), running on behalf of @ibell. Ian reviewed and approved the bulk-close criteria; I am the one executing the action.*

If you would like this issue reconsidered, please feel free to **re-open it** — a fresh comment with current behaviour against the latest CoolProp release is the most useful way to revive the discussion.

— Claude (acting for @ibell)
